HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc

```diff
@@ -319,6 +350,8 @@ void HLTScoutingEgammaProducer::fillDescriptions(edm::ConfigurationDescriptions&
   desc.add<double>("egammaPtCut", 4.0);
   desc.add<double>("egammaEtaCut", 2.5);
   desc.add<double>("egammaHoverECut", 1.0);
+  desc.add<std::vector<double>>("egammaSigmaIEtaIEtaCut", {99999.0, 99999.0});
+  desc.add<std::vector<double>>("absEtaLowEdges", {0.0, 1.479});
```

Comment:
From a quick look, it looks more intuitive to me to have `egammaSigmaIEtaIEtaCutMaxAbsEta`, such that
```python
 egammaSigmaIEtaIEtaCut          = {0.6 , -1.0 , 0.4 }
 egammaSigmaIEtaIEtaCutMaxAbsEta = {1.5 ,  2.5 , 3.0 }
```
so that `sigmaIEtaIEta > 0.6` if `|eta| < 1.5`, etc. No?

Reply:
Yes. I would normally do it this way. I was following the standard Egamma cut strategy as in https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Egamma/plugins/HLTEgammaGenericQuadraticEtaFilter.cc#L57. 

I would change it to reflect your suggestion. Although I would leave the name as just "absEtaBinMax" so that if we introduce some other variable with a different barrel and end-cap cut then we wouldn't need to change the name for this variable.

Reply:
Please let me know if you feel like the naming for the eta binning should be different to make it more intuitive.